src/containers/stories/story.container.jsx

```diff
@@ -0,0 +1,52 @@
+/* @flow */
+import type { State } from "../../state/state.type.js";
+import type { Action } from "./story.actions.js";
+import type { StoryState, Story as StoryType } from "./stories.type";
+
+import { connect } from "react-redux";
+import React, { Component } from "react";
+import { requestStory } from "./story.actions.js";
+import Typography from "@material-ui/core/Typography";
+
+import LoadingAnimation from "./../../../src/components/loading-animation";
+
+type Props = {
+  story: StoryState,
+  dispatch: Action => void
+};
+
+class StoryContainer extends Component<Props> {
+  componentDidMount = () => {
+    this.props.dispatch(requestStory(this.props.match.params))
```

Comment:
Ici ce que tu fais c'est des que le component est rendu, tu va demande la story au backend en passant par redux (c'est exactement ce qu'il faut faire). Ce qu'il va se passer:
1- requestStory fonction est execute, elle va retourne un object en fonction de l'id qu'elle recoit, disons 1234 pour l'example: {type: "REQUEST_ONE_STORY",id: 1234})
2- Cet object vas etre 'dispatcher': this.props.dispatch({type: "REQUEST_ONE_STORY",id: 1234}). Dispatch ca veux juste dire que ca va notifier les reducers que il faut faire quelque chose
3- Le reducer va voir: 'oh une action viens d'arriver, voyons ce que il faut que je fasse'
4- Oh, l'action s'appelle REQUEST_ONE_STORY, je sais ce que je dois faire. Il faut que je j'execute fx (qui va faire la requete a l'API). 
5- En attendant que ka requete sois terminee, en tant que bon reducer je vais update mon state a mon state courant et loading: true
6- Oh, l'api reviens, je vais update mon state avec la valeur retourne.
7- Le react composant StoryContainer est abone a ce state, il va donc s'updater tout seul (car il recois story en props. Dans react, des que les props change le component re-execute render)
8- Bam, la story s'affiche :)

Un petit truc avant de faire tout sa, verifie que this.props.match.params est pas null car sinon faire tout sa ne sers a rien haha

donc rajoute
```
// Equivalent a const params = this.props.match.params.;
const {params} = this.props.match;
if (params != null) {
  this.props.dispatch(requestStory(params))
}
```